src/lib.rs

```diff
@@ -706,4 +757,38 @@ mod tests {
 
         assert!(!pk.verify_correct_decryption(proof, ciphertext, fake_decryption));
     }
+
+    #[test]
+    fn test_add_of_ciphertext_and_plaintext() {
+        let mut csprng = OsRng;
+        let sk = SecretKey::new(&mut csprng);
+        let pk = PublicKey::from(&sk);
+
+        let plaintext = RistrettoPoint::random(&mut csprng);
+        let ciphertext = pk.encrypt(plaintext);
+        let plaintext2 = RistrettoPoint::random(&mut csprng);
+
+        println!(
+            "1: {:?}, 2: {:?}, 3: {:?}",
+            sk.decrypt(plaintext2 + ciphertext),
+            sk.decrypt(ciphertext + plaintext2),
+            plaintext + plaintext2
+        );
```

Comment:
This should be removed.  To maintain the consistency of not printing anything in the tests. This is the only change I see necessary.

Reply:
Thanks. I've already removed it.